src/proto/h1/encode.rs

```diff
@@ -136,6 +163,53 @@ impl Encoder {
         EncodedBuf { kind }
     }
 
+    pub(crate) fn encode_trailers<B>(
+        &self,
+        trailers: HeaderMap,
+        title_case_headers: bool,
+    ) -> Option<EncodedBuf<B>> {
+        match &self.kind {
+            Kind::Chunked(Some(ref allowed_trailer_fields)) => {
+                let allowed_trailer_field_map = allowed_trailer_field_map(&allowed_trailer_fields);
+
+                let mut cur_name = None;
+                let mut allowed_trailers = HeaderMap::new();
+
+                for (opt_name, value) in trailers.into_iter() {
```

Comment:
Thanks! Looks good to me. `trailers.into_iter()` can be just `trailers` here.

Refs: https://doc.rust-lang.org/std/iter/index.html#for-loops-and-intoiterator

Reply:
oops! fixed in 7a8eb3381387aed4f126a7c5fc366b8310b05525